### PR TITLE
E-1298 feat: remove ability to upload selfie from hdd and add mobile suggestion

### PR DESCRIPTION
**Pull request by ifiri**

### CODE REVIEW

File: src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/UploadSide.vue [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <div>
     <upload-header :document-type="documentType" />
     <drop-area
+      v-if="!isSelfie"
```
Comment (dudiq) [inline src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/UploadSide.vue:5]: no need to check selfie it or not, just remove drop-area will be enough

File: src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/UploadSide.vue [hunk 2]
```diff
@@ -20,6 +21,7 @@
         @file-remove="onFileRemove"
       />
     </drop-area>
+    <mobile-suggestions v-if="isSelfie" />
```
Comment (dudiq) [inline src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/UploadSide.vue:24]: same story, no need to check

File: src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/UploadSide.vue (hunk 1 above)
Comment (ifiri) [reply src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/UploadSide.vue:5]: done

File: src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/UploadSide.vue (hunk 2 above)
Comment (ifiri) [reply src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/UploadSide.vue:24]: done

Comment (dudiq) [reply src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/UploadSide.vue:24]: where is commit?

Comment (ifiri) [reply src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/UploadSide.vue]: done

File: src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js [hunk 3]
```diff
@@ -20,39 +20,69 @@ describe('UploadVideo > UploadSide', () => {
...
-  });
-
   const createWrapper = options => {
     return shallowMount(UploadSide, {
       localVue,
       i18n,
       sync: false,
       propsData: {
-        documentType: DOC_TYPES.SELFIE,
+        documentType: DOC_TYPES.PASSPORT,
```
Comment (dudiq) [inline src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js:29]: upload side only for selfie, no need to set passport

File: src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js [hunk 4]
```diff
@@ -20,39 +20,69 @@ describe('UploadVideo > UploadSide', () => {
...
       },
       ...options,
     });
   };
 
   const emitUpload = async () => {
-    wrapper.find('drop-area-stub').vm.$emit('change', [file]);
+    wrapper = createWrapper({
```
Comment (dudiq) [inline src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js:36]: if you have method `emitUpload`, it should emit, not create new wrapper instance.
create wrapper in `beforeEach` method

File: src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js [hunk 5]
```diff
@@ -20,39 +20,69 @@ describe('UploadVideo > UploadSide', () => {
...
 
   describe('render', () => {
     it('should correctly render component', () => {
+      expect.assertions(2);
+
       expect(wrapper.name()).toBe('UploadSide');
       expect(wrapper.html()).toMatchSnapshot();
     });
+
+    it('should not be able to upload file from pc if selfie', () => {
```
Comment (dudiq) [inline src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js:61]: no need this test anymore

File: src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js [hunk 6]
```diff
@@ -117,29 +147,14 @@ describe('UploadVideo > UploadSide', () => {
       expect(wrapper.emitted().confirm).toBeUndefined();
     });
 
-    it('should show error when upload', async () => {
```
Comment (dudiq) [inline src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js:120]: and how we process errors now, if we have they?

File: src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js (hunk 3 above)
Comment (ifiri) [reply src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js:29]: done

File: src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js (hunk 4 above)
Comment (ifiri) [reply src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js:36]: done

File: src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js (hunk 5 above)
Comment (ifiri) [reply src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js:61]: done

File: src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js (hunk 6 above)
Comment (ifiri) [reply src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js:120]: done

did extend state label

Comment (dudiq) [reply src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js:120]: green error, are you sure?
image

Comment (ifiri) [reply src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js:120]: really, text so small that color wasn't noticed lol

Comment (ifiri) [reply src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js:120]: done

**Pull request merged**